.circleci/config.yml

```diff
@@ -155,21 +155,25 @@ jobs:
             cd packages/graphql_flutter
 
             # patch version
-            ~/yq w pubspec.yaml version $CIRCLE_TAG > patched_pubspec.yaml
+            ~/yq w pubspec.yaml version ${CIRCLE_TAG%"@beta"} > patched_pubspec.yaml
             rm pubspec.yaml
             mv patched_pubspec.yaml pubspec.yaml
 
             # patch graphql dependency version
-            ~/yq w pubspec.yaml dependencies.graphql ^$CIRCLE_TAG > patched_pubspec.yaml
+            ~/yq w pubspec.yaml dependencies.graphql ^${CIRCLE_TAG%"@beta"} > patched_pubspec.yaml
```

Comment:
will we need to change this when merging `beta` into `master`?

Reply:
Nope! It's bash for: `optionally strip "@beta"`.